packages/admin-ui/src/filters/filter-radio.tsx

```diff
@@ -11,6 +11,8 @@ export const FilterRadio = (props: FilterRadioProps) => {
       ':after': style.checkmark,
       ':checked': style.checked,
       ...style.regular,
+      ariaChecked: undefined,
```

Comment:
is this right?

Reply:
Yes, this radio is acting almost as an icon, it's already encapsulated by a combobox item with ariachecked defined